acto/input/test_generators/stateful_set.py

```diff
@@ -64,7 +64,7 @@ def replicas_tests(schema: IntegerSchema) -> list[TestCase]:
 @test_generator(
     property_name="statefulSetUpdateStrategy", priority=Priority.SEMANTIC
```

Comment:
@LouHaina I realized that the root cause of this problem is the `property_name` arg. It should be "updateStrategy", corresponding to the https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/#updating-statefulsets

Since we had a wrong name, this test generator is mistakenly matched to a wrong field in tidb-operator.

Could you change the fix to change the `property_name`? With that fixed, the rest of the fix should not be needed.

Reply:
I have updated the property_name to "updateStrategy" without the previously proposed fixes, and pushed the updates